tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/ExtensionsMetadataGenerator.cs

```diff
@@ -90,6 +90,11 @@ public static bool IsWebJobsStartupAttributeType(TypeReference attributeType, Co
                     return true;
                 }
 
+                if (currentAttributeType.FullName.StartsWith("System", StringComparison.OrdinalIgnoreCase))
```

Comment:
Which assembly, and type did you hit this with? I'm curious whether there's some class of types that we cannot call `Resolve()` on. There may be a better check than just whether they begin with "System".

Reply:
Good day, I have faced exactly same problem. For me it hanged on `System.ComponentModel.DataAnnotations.KeyAttribute`. It took me a while to figure out what is going on. Is there any workaround for this until this fix is released?